com.endlessm.HackSoundServer.appdata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.endlessm.HackSoundServer.desktop</id>
```

Comment:
I don't think HackSoundServer has a desktop file... Will that have any impact on how the appdata gets used down the line? Should we add one?

Reply:
I don't think the desktop suffix in the ID has to do with having a desktop file or not. I noticed that AppStream does use it for apps and we had problems before with a couple of apps that didn't have that ID and were thus not being matched correctly to the ones we have in the proxy apps.

Reply:
OK then, let's get these in.